client/src/tests/photos.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+import Photos from '../components/photos.jsx';
+
+const wrapper = shallow(<Photos />);
```

Comment:
Grats on getting enzyme running, I don't understand it enough myself but your code looks nice.

lgtmafaik!